agent-core/src/main/java/com/newrelic/agent/android/stats/TicToc.java

```diff
@@ -17,11 +17,11 @@ private static enum State {
 
     public void tic() {
```

Comment:
Could you please return the TicToc instance here? ```public TicTok tic() { return this; }```

Reply:
Fixed